test/integration/users.test.js

```diff
@@ -585,6 +575,39 @@ describe("Users", function () {
     });
   });
 
+  describe("GET /users/?id", function () {
+    beforeEach(async function () {
+      await addJoinData(joinData(userId)[0]);
+    });
+    it("Should return given user by id", function (done) {
+      chai
+        .request(app)
+        .get(`/users/?id`)
```

Comment:
I think the test cases are failing as no Id is being passed, @heyrandhir @RitikJaiswal75 correct me if wrong.